app/lib/common/widget/SideMenu.dart

```diff
@@ -0,0 +1,404 @@
+// ignore_for_file: prefer_const_constructors
+
+import 'dart:typed_data';
+
+import 'package:effektio/common/store/Colors.dart';
+import 'package:effektio/screens/UserScreens/SocialProfile.dart';
+import 'package:effektio_flutter_sdk/effektio_flutter_sdk.dart';
+import 'package:effektio_flutter_sdk/effektio_flutter_sdk_ffi.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class SideDrawer extends StatefulWidget {
+  const SideDrawer({Key? key, required this.client}) : super(key: key);
+  final Future<Client> client;
+  @override
+  State<SideDrawer> createState() => _SideDrawerState();
+}
+
+class _SideDrawerState extends State<SideDrawer> {
+  late Future<String> name;
+  late Future<String> username;
+  late Future<List<int>> avatar;
+  bool isGuest = false;
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Drawer(
+      backgroundColor: AppColors.textFieldColor,
```

Comment:
We can customize AppColors, infact AppColors is already defined as custom class with custom colors. We can just name as a seperate entity in AppColors.dart. This was already defined in provided screens, I didn't changed it.

Reply:
I know we had that before, I was referring to replacing that with a proper https://api.flutter.dev/flutter/material/ThemeData-class.html implementation rather than picking random names that just currently fit the colorscheme... Like if we now changed the textFieldColor, this would also change the drawer background? That's just wrong.

Reply:
I'll replace that with ThemeData class in next commit.